tests/visco_plastic_adiabatic_pressure.prm

```diff
@@ -0,0 +1,107 @@
+# A test for using adiabatic pressure in creep viscosity calculation for
+# the visco_plastic material model.
+
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 1
+set Output directory                       = visco_plastic_adiabatic_pressure
```

Comment:
The name of this test is a bit ambiguous. I would suggest renaming to something like `visco_plastic_adiabatic_pressure_in_viscous_creep`?

You can change the name of files with `git mv original_file_name new_file_name`